src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -259,20 +259,21 @@ export class SuggestController implements IEditorContribution {
 			this.editor.pushUndoStop();
 		}
 
-		if (Array.isArray(suggestion.additionalTextEdits)) {
-			this.editor.executeEdits('suggestController.additionalTextEdits', suggestion.additionalTextEdits.map(edit => EditOperation.replace(Range.lift(edit.range), edit.text)));
-		}
+		// compute overwrite[Before|After] deltas BEFORE applying extra edits
+		const info = this.getOverwriteInfo(item, Boolean(flags & InsertFlags.AlternativeOverwriteConfig));
 
 		// keep item in memory
 		this._memoryService.memorize(model, this.editor.getPosition(), item);
```

Comment:
Why does this line also need to move? It looks like even in 1.40 it was after the `executeEdits`.

Reply:
Yeah - that's a non minimal edit and fixes another (un-filed) issue where the memory service would also use the wrong editor position.

Reply:
LGTM